services/CustomFees.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import "BasicTypes.proto";
+import "google/protobuf/wrappers.proto";
+
+/* A fraction of the units to be collected from the payer of a CryptoTransfer 
+that transfers units of a token. The amount charged will never be
+less than the stated minimumUnitsToCollect, and never greater than 
+the stated maximumUnitsToCollect (if given). */
+message FractionalFee {
+  Fraction fractionOfUnitsToCollect = 1; // percentage of the Tokens involved in the transfer to be charged as fee
+  uint64 minimumUnitsToCollect = 2; // minimum token units to be charged as fee
+  google.protobuf.UInt64Value maximumUnitsToCollect = 3; // maximum token units to be charged as fee
```

Comment:
Having `OfUnitsToCollect` appended to every field seems redundant. Fees by their nature are always something to be collected. And I think `amount` is a more appropriate term. Or you can just drop the repeated suffix since it's already within the context of `FractionalFee` wrapper to make them `fraction`, `minimum` and `maximum`.

Reply:
Can we just use `uint64` for `maximumUnitsToCollect` too? value 0 means no max set.

Reply:
Since we require fully replacing the `CustomFees` field on an update, this makes sense. 👍